samples/apps/auth-api-webapp-react/README.md

```diff
@@ -9,19 +9,19 @@
 
 ## Running the sample
 
-1. You will need an [Open AI Key](https://openai.com/api/) or
+1. You will need an [Open AI Key](https://platform.openai.com/account/api-keys) or
    [Azure Open AI Service key](https://learn.microsoft.com/azure/cognitive-services/openai/quickstart)
    for this sample
-2. Ensure the service API is already running `http://localhost:7071`. If not learn
+2. Ensure the KernelHttpServer sample is already running at `http://localhost:7071`. If not, follow the steps 
    how to start it [here](../../dotnet/KernelHttpServer/README.md).
 3. You will also need to
    [register your application](https://learn.microsoft.com/azure/active-directory/develop/quickstart-register-app)
    in the Azure Portal. Follow the steps to register your app
    [here](https://learn.microsoft.com/azure/active-directory/develop/quickstart-register-app).
     - Select **`Single-page application (SPA)`** as platform type, and the Redirect URI will be **`http://localhost:3000`**
-    - It is recommended you use the **`Personal Microsoft accounts`** account type for this sample.
+    - Select **`Personal Microsoft accounts only`** as supported account types for this sample
 4. Once registered, copy the **Application (client) ID** from the Azure Portal and paste
-   the GUID into the **[.env](.env)** file next to `REACT_APP_GRAPH_CLIENT_ID=` (first line of the .env file).
+   the GUID into the new **[.env](.env)** file created next to `REACT_APP_GRAPH_CLIENT_ID=` (first line of the .env file).
```

Comment:
"into the new .env file create next to...."

Sounds to me like we've just created a new .env file for them. Is that accurate?

Reply:
Good point, missed that when moved from .env.development back to .env.